README.md

```diff
@@ -68,7 +68,10 @@ and get the following JSON as the output:
       " Battalion S4"
     ]
   },
-  "intermediate_rater": "1LT X is #2 of the 20 Lieutenants I senior rated. He is an asset for the future and will progress further in his military career. Keep assigning him to demanding position and select him for the Military Police Captains Career Course now. Promote ahead of peers to Captain and select him for the next Company Command."
+  "intermediate_rater": "1LT X is #2 of the 20 Lieutenants I senior rated. He is an asset for the future and will progress further in his military career. Keep assigning him to demanding position and select him for the Military Police Captains Career Course now. Promote ahead of peers to Captain and select him for the next Company Command.",
+  "performance": "PROFICIENT",
```

Comment:
Can we nest this and `referred` under the `rater` key since those blocks come from the rater?